config/src/config/mod.rs

```diff
@@ -320,6 +320,27 @@ impl NodeConfig {
         self.set_data_dir(test.temp_dir().unwrap().to_path_buf());
         self.test = Some(test);
     }
+
+    #[cfg(any(test, feature = "fuzzing"))]
+    pub fn default_for_public_full_node() -> Self {
+        let contents = std::include_str!("test_data/public_full_node.yaml");
+        let path = "default_for_public_full_node";
```

Comment:
any reason we don't just inline path into the error message?

Reply:
wanted to avoid the weird nested new line that would come as a result :P